nwbinspector/checks/tables.py

```diff
@@ -134,8 +135,15 @@ def check_column_binary_capability(table: DynamicTable, nelems: int = 200):
 
 
 @register_check(importance=Importance.BEST_PRACTICE_SUGGESTION, neurodata_type=DynamicTable)
-def check_single_row(table: DynamicTable):
-    """Check if DynamicTable has only a single row; may be better represented by another data type."""
+def check_single_row(table: DynamicTable, exclude_names: List[str] = ["Units", "electrodes"]):
```

Comment:
I wanted this to be `exclude_types = [pynwb.Units, pynwb.ElectrodeTable]` with `isinstance` logic checking if the `table` was a subtype of that exclusion. While this works perfectly fine for the `Units`, apparently the `ElectrodeTable` is not actually a subtype of a DynamicTable but rather a function that returns a DynamicTable of a particular structure.

Reply:
yes, there is no ElectrodeTable neurodata type. It's an oversight in the schema.

Reply:
can you create two lists, one for exclude neurodata types and one for exclude names?

Reply:
Done